wesgi/tests.py

```diff
@@ -99,7 +106,7 @@ def side_effect(*args):
         data = mw._process_include('level-1<esi:include src="http://www.example.com"/>-after')
         self.assertEquals(data, 'level-1-2-3-4-5-6-7-8-9-10-11-last-after')
         self.assertEquals(mock.call_count, 11)
-        self.assertEquals(mock.call_args, (('http://www.example.com/10', ), ))
+        self.assertEquals(str(mock.call_args), "call('http://www.example.com/10', headers={})")
```

Comment:
hmm, I didn't know you could call str() on call_args. looks easier, but I think I've been too burnt with doctests in the past to like it...

just a the gripes of an old man, please don't change the PR for this